splink/internals/database_api.py

```diff
@@ -367,5 +391,66 @@ def delete_tables_created_by_splink_from_db(self):
             ):
                 splink_df.drop_table_from_database_and_remove_from_cache()
 
+    def _bind_templated_alias_to_physical(self, templated: str, physical: str) -> None:
+        """Default: try TEMP VIEW, fall back to a normal VIEW."""
+        try:
+            self._create_or_replace_temp_view(templated, physical)
+        except NotImplementedError:
+            self._create_or_replace_view(templated, physical)
+
+    def _unbind_templated_alias_from_physical(self, templated: str) -> None:
```

Comment:
We could have just used normal views, but a theoretical advantage of temporary ones is it's less likely for tests running in parallel to conflict with one another. Whether this would be a real-world I'm not sure